source/_data/glossary.yml

```
@@ -271,8 +271,25 @@
   link: /docs/blueprint/selectors/
 
 - term: Service
-  definition: >-
-    Services are called to perform actions.
+  definition: |-
+    The term service has 2 meanings in Home Assistant:
+
+    **The information service**
+    E.g. the municipal waste management service that provides entities for organic, paper, and packaging waste. In terms of functionality, the information service is a device. It is called *service* to avoid confusion as it does not come with a piece of hardware.
+
+    **The software function that interacts with targets to make something happen**
+    A service carries out one specific task: e.g. turn on the light in the living room or send a notification to a mobile phone. 
+
+    A service has targets and data. Service targets are: area, device, and entity. Service data carries the information required to define the desired state change in the target. E.g. the target ID, together with brightness 150 and rgb color [255,0,0], or the message “Your coffee is ready”. 
+
+    Services can be called by actions, a dashboard, or via voice command. 
```

A bit inconsistent in plural vs singular. I also think an action is more a part of something else, while a dashboard or voice command is more something a user uses.

How about:

```suggestion
    Services can be used in, for example, automation, scripts, dashboards, or voice commands to control your home.
```

---

Love the solution. I felt something was amiss but could not put my finger on it.